core/src/test/java/io/grpc/util/RoundRobinLoadBalancerTest.java

```diff
@@ -137,20 +141,25 @@ public Subchannel answer(InvocationOnMock invocation) throws Throwable {
   }
 
   @After
+  @SuppressWarnings({"deprecation", "unchecked"})
   public void tearDown() throws Exception {
     verifyNoMoreInteractions(mockArgs);
+    verify(mockHelper, never()).createSubchannel(any(List.class), any(Attributes.class));
```

Comment:
nit: `ArgumentMatchers.<List<EAG>>any()` does not need suppress  unchecked.

Reply:
Done.